lib/linguist/heuristics.rb

```diff
@@ -73,7 +73,7 @@ def call(data)
       if (/@(interface|class|protocol|property|end|synchronised|selector|implementation)\b/.match(data))
```

Comment:
*sighs* I feared that might happen.. only thing we can do for the time being is anchor to the start of the line and add a skip for white-space only. this will still match under certain conditions even when it shouldn't but without teaching linguist how to scan comments to exclude them it's the best we can do.